run

```diff
@@ -766,6 +773,7 @@ if [ -s $outfile ] && tar -tf $outfile &>/dev/null; then
     if [ -r "$OUTPUT/exit-code" ]; then
         exit $(<"$OUTPUT/exit-code")
     else
+        clean
```

Comment:
I wonder if we need it. Would the trap above would do the clean up?

Reply:
Of course:)